### PR TITLE
Mac Catalyst App: Make window small again

### PR DESCRIPTION
**Pull request by nalinbhardwaj**
Custom AppDelegate that sizes the window to a fixed ~mobile size :)

Screenshot 2024-03-07 at 2 34 29 PM

### CODE REVIEW

File: apps/daimo-mobile/package.json [hunk 1]
```diff
@@ -3,12 +3,12 @@
...
-    "build:sim": "eas build -p ios --profile=sim --non-interactive",
-    "build:dev": "eas build -p ios --profile=dev --non-interactive",
+    "build:sim-ios": "eas build -p ios --profile=sim --non-interactive",
+    "build:dev-ios": "eas build -p ios --profile=dev --non-interactive",
     "build:dev-android": "eas build -p android --profile=dev --non-interactive",
     "build:prod": "eas build -p all --profile=production --auto-submit --non-interactive",
-    "submit:prod": "eas submit --platform ios --profile=production",
-    "submit:prod-android": "eas submit --platform android --profile=production",
+    "build:prod-ios": "eas build -p ios --profile=production --auto-submit --non-interactive",
+    "build:prod-android": "eas build -p android --profile=production --auto-submit --non-interactive",
```
Comment (dcposch) [inline apps/daimo-mobile/package.json:11]: 👍

File: packages/daimo-expo-app-delegate/README.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
+# @daimo/expo-app-delegate
+
+Daimo's custom AppDelegate
```
Comment (dcposch) [inline packages/daimo-expo-app-delegate/README.md:3]: what does an AppDelegate do?

i'd add a sentence or two here explaining purpose / goals

File: packages/daimo-expo-app-delegate/ios/DaimoAppDelegate.swift [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+      return false
+    #endif
+  }
+
+  public func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey : Any]? = nil) -> Bool {
+    
+    if self.isOnMac() {
+      UIApplication.shared.connectedScenes.compactMap { $0 as? UIWindowScene }.forEach { windowScene in
+        windowScene.sizeRestrictions?.minimumSize = CGSize(width: 540, height: 900)
+        windowScene.sizeRestrictions?.maximumSize = CGSize(width: 540, height: 900)
```
Comment (dcposch) [inline packages/daimo-expo-app-delegate/ios/DaimoAppDelegate.swift:22]: so this is the money line: fixed size, nonresizeable?

**Review by dcposch — APPROVED** [review]
LGTM after comments

File: packages/daimo-expo-app-delegate/ios/DaimoAppDelegate.swift (hunk 3 above)
Comment (nalinbhardwaj) [reply packages/daimo-expo-app-delegate/ios/DaimoAppDelegate.swift:22]: yep

File: packages/daimo-expo-app-delegate/README.md (hunk 2 above)
Comment (nalinbhardwaj) [reply packages/daimo-expo-app-delegate/README.md]: done

**Pull request merged**